examples/Workflow/README.md

```diff
@@ -11,105 +11,113 @@ This Dapr workflow example shows how to create a Dapr workflow (`Workflow`) and
+curl -i -X POST http://localhost:3500/v1.0-alpha1/workflows/dapr/OrderProcessingWorkflow/1234/start `
   -H "Content-Type: application/json" `
-  -d '{"name": "Paperclips", "totalCost": 99.95, "quantity": 1}'
+  -d '{ "input" : {"Name": "Paperclips", "TotalCost": 99.95, "Quantity": 1}}'
 ```
 
-If successful, you should see a response like the following, which contains a `Location` header pointing to a status endpoint for the workflow that was created with a randomly generated 8-digit ID:
+If successful, you should see a response like the following: 
 
-```http
-HTTP/1.1 202 Accepted
-Content-Length: 0
-Date: Tue, 24 Jan 2023 00:02:02 GMT
-Server: Kestrel
-Location: http://localhost:10080/orders/cdcce425
+```json
+{"instance_id":"1234"}
 ```
 
-Next, send an HTTP request to the URL in the `Location` header in the previous HTTP response, like in the following example:
+Next, send an HTTP request to get the status of the workflow that was started:
 
 ```bash
-curl -i http://localhost:10080/orders/cdcce425
+curl -i -X GET http://localhost:3500/v1.0-alpha1/workflows/dapr/OrderProcessingWorkflow/1234
 ```
 
-If the workflow has completed running, you should see the following output (formatted for readability):
-
-```http
-HTTP/1.1 200 OK
-Content-Type: application/json; charset=utf-8
-Date: Tue, 24 Jan 2023 00:10:53 GMT
-Server: Kestrel
-Transfer-Encoding: chunked
+The workflow is designed to take several seconds to complete. If the workflow hasn't completed yet when you issue the previous command, you should see the following JSON response (formatted for readability):
 
+```json
 {
-    "details": {
-        "name": "Paperclips",
-        "quantity": 1,
-        "totalCost": 99.95
-    },
-    "result": {
-        "processed": true
-    },
-    "status": "Completed"
+  "WFInfo": {
+    "instance_id": "1234"
+  },
+  "start_time": "2023-02-02T23:34:53Z",
+  "metadata": {
+    "dapr.workflow.custom_status": "",
+    "dapr.workflow.input": "{\"Name\":\"Paperclips\",\"Quantity\":1,\"TotalCost\":99.95}",
+    "dapr.workflow.last_updated": "2023-02-02T23:35:07Z",
+    "dapr.workflow.name": "OrderProcessingWorkflow",
+    "dapr.workflow.output": "{\"Processed\":true}",
+    "dapr.workflow.runtime_status": "RUNNING"
+  }
 }
 ```
 
-If the workflow hasn't completed yet, you might instead see the following:
-
-```http
-HTTP/1.1 202 Accepted
-Content-Type: application/json; charset=utf-8
-Date: Tue, 24 Jan 2023 00:17:49 GMT
-Location: http://localhost:10080/orders/cdcce425
-Server: Kestrel
-Transfer-Encoding: chunked
+Once the workflow has completed running, you should see the following output, indicating that it has reached the "COMPLETED" status:
 
+```json
 {
-    "details": {
-        "name": "Paperclips",
-        "quantity": 1,
-        "totalCost": 99.95
-    },
-    "status": "Running"
+  "WFInfo": {
+    "instance_id": "1234"
+  },
+  "start_time": "2023-02-02T23:34:53Z",
+  "metadata": {
+    "dapr.workflow.custom_status": "",
```

Comment:
should we fulfill the `custom_status` field with something meaningful ?

Reply:
I think we can consider this for a future update.